common/test/acceptance/tests/helpers.py

```diff
@@ -96,6 +96,16 @@ def load_data_str(rel_path):
         return data_file.read()
 
 
+def remove_file(filename):
+    """
+    Remove a file if it exists
+    """
+    try:
+        os.remove(filename)
+    except OSError:
```

Comment:
I'm not sure what other conditions might raise an OSError.
Another alternative pattern is to do something like this:
```
if os.path.exists(filename):
  os.remove(filename)
```
This pattern is already in our codebase [here](../../tree/master/common/djangoapps/status/tests.py#L81)